packages/functional-tests/tests/stub.spec.ts

```diff
@@ -1,7 +0,0 @@
-import { test } from '../lib/fixtures/standard';
-
-test('stub', async ({ page, testAccountTracker }) => {
```

Comment:
reminder (just in case): update the [README](https://github.com/mozilla/fxa/blob/main/packages/functional-tests/README.md#creating-new-tests) with this removal.

Reply:
yes, good catch.